jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/kubernetes/KubernetesClientUtilTest.java

```diff
@@ -163,4 +162,58 @@ void getPodStatusMessagePostfix_whenActionIsError_shouldReturnErrorMessage() {
     assertThat(messagePostfix).isEqualTo(": Error");
   }
 
+  @Test
+  void getPodCondition_whenPodConditionIsReadyAndTrue_shouldReturnReady() {
+    // Given
+    Pod pod = new Pod();
```

Comment:
Could you please check if you can simplify this by using `new PodBuilder().withNewStatus()... .build()`